drivers/ethos/ethos.c

```diff
@@ -33,7 +33,7 @@
 #include "net/eui64.h"
 #include "net/ethernet.h"
 
-#ifdef USE_ETHOS_FOR_STDIO
+#ifdef MODULE_STDIO_ETHOS
```

Comment:
It would be nice to have a warning if `USE_ETHOS_FOR_STDIO` is still set saying "Deprecated: you must now use `USEMODULE+=stdio_ethos`"

Reply:
warnings are treated as errors, so I just added an error message using `#error` macro.

Reply:
fixed